examples/service-per-pod/test.sh

```diff
@@ -24,3 +27,30 @@ until [[ "$(kubectl get svc nginx-2 -o 'jsonpath={.spec.selector.pod-name}')" ==
 
 echo "Wait for pod-name label..."
 until [[ "$(kubectl get pod nginx-2 -o 'jsonpath={.metadata.labels.pod-name}')" == "nginx-2" ]]; do sleep 1; done
+
+echo "Remove annotation to opt out of service-per-pod without deleting the StatefulSet..."
+kubectl annotate statefulset nginx service-per-pod-label-
```

Comment:
For `kubectl annotate`, the syntax `annotation-` means "remove this annotation", while `annotation=` means "set this annotation to the empty string". Since our decorator selector only requires this label to exist, we need to remove it entirely to prevent a match.

Reply:
noted, never know this. thanks!